packages/x-charts/src/ChartsSurface.tsx

```diff
@@ -9,14 +9,25 @@ type ViewBox = {
   height?: number;
 };
 export interface ChartsSurfaceProps {
+  /**
+   * The width of the chart in px.
+   */
   width: number;
+  /**
+   * The height of the chart in px.
+   */
   height: number;
   viewBox?: ViewBox;
   className?: string;
   title?: string;
   desc?: string;
   sx?: SxProps<Theme>;
   children?: React.ReactNode;
+  /**
+   * If true, the charts will not listen to the mouse move event.
+   * It might breaks interactive features, but save some computation power.
+   * @default false
+   */
```

Comment:
```suggestion
  /**
   * If `true`, the charts will not listen to the mouse move event.
   * It might break interactive features, but is improving performance.
   * @default false
   */
```